Signal/src/Loki/Utilities/LokiPushNotificationManager.swift

```diff
@@ -13,7 +13,9 @@ final class LokiPushNotificationManager : NSObject {
     func register(with token: Data) {
         let hexEncodedToken = token.map { String(format: "%02.2hhx", $0) }.joined()
         let oldToken = UserDefaults.standard.string(forKey: "deviceToken")
-        if (hexEncodedToken == oldToken) {
+        let lastUploadTime = UserDefaults.standard.integer(forKey: "lastUploadTime")
```

Comment:
`TimeInterval` is just a type alias for `Double` so there's no need to convert to integer and back

Reply:
Also, this is the global user defaults so this key is a bit generic